### PR TITLE
About Page

### PR DESCRIPTION
**Pull request by kirakira0**
image

image

### CODE REVIEW

File: client/src/components/About/About.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+a {
+  color: lightgray;
+  text-decoration: none;
+  transition: 0.5s;
+}
+ 
+a:hover {
+  color: black;
+  transition: 0.5s; 
+}
```
Comment (rob-marcus) [inline client/src/components/About/About.css:46]: Minor nit: new line.

File: client/src/components/About/Profile.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+ 
+const Profile = props => {
+ 
+  function getProfileText() {
+    if (props.type === 'intern') {
+      return (
+        <div className="profile">
+          <h3>{props.name}</h3>
+          <p>Google STEP Intern</p>
+          <p>Github: {props.github}</p>
```
Comment (rob-marcus) [inline client/src/components/About/Profile.js:11]: Might be nice to make this a link

File: client/src/components/About/About.js [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+          our gratitude and appreciation to you both for your tireless advice 
+          and encouragement. We have all grown as engineers as a result of 
+          your wisdom and support, and we thank you sincerely for it.
+        </p>
+      </div>
+    </div>
+  );
+};
+ 
+export default About;
```
Comment (rob-marcus) [inline client/src/components/About/About.js:131]: Minor nit: new line

File: client/src/constants/header_constants.js [hunk 4]
```diff
@@ -1,4 +1,5 @@
 const HEADERS = [
+  {route: '/about', text: 'About'},
```
Comment (scarletnguyen13) [inline client/src/constants/header_constants.js:2]: nit: `{ route: '/about', text: 'About' }`

**Pull request merged**